src/handlers/dns/DnsStreamHandler.h

```diff
@@ -127,6 +130,7 @@ class DnsMetricsBucket final : public visor::AbstractMetricsBucket
         , _dns_topQname3("dns", "qname", {"top_qname3"}, "Top QNAMES, aggregated at a depth of three labels")
         , _dns_topNX("dns", "qname", {"top_nxdomain"}, "Top QNAMES with result code NXDOMAIN")
         , _dns_topREFUSED("dns", "qname", {"top_refused"}, "Top QNAMES with result code REFUSED")
+        , _dns_topSizedQnameResp("dns", "qname", {"top_qname_by_response"}, "Top QNAMES by response size")
```

Comment:
i think we need `top_qname_by_response_size` ... possibly `top_qname_by_resp_size` if we don't have another precedent?

Reply:
yeah it's ok, the internal name i don't care as much about since it's not a public interface, we can always refactor it later

Reply:
cool